yarn.lock

```diff
@@ -19437,7 +19451,7 @@ sequencify@~0.0.7:
   resolved "https://registry.yarnpkg.com/sequencify/-/sequencify-0.0.7.tgz#90cff19d02e07027fd767f5ead3e7b95d1e7380c"
   integrity sha1-kM/xnQLgcCf9dn9erT57ldHnOAw=
 
-serialize-javascript@^1.4.0, serialize-javascript@^2.1.1:
+serialize-javascript@^1.4.0, serialize-javascript@^2.1.1, serialize-javascript@^4.0.0:
   version "2.1.2"
```

Comment:
This doesn't look right -- looks like we have a resolution in `package.json` that is causing this. I added this in #64936, and in the commit message inside the PR I explained my rationale for forcing `1.4.0` to  resolve to`2.1.1`:
> The PR this backports actually updated the resolution from 1.7.0 to
2.1.1. This commit updates the resolution from 1.5.0 to 2.1.1.
I checked the changelog and it doesn't appear that there are any
breaking changes between 1.5.0 and 1.7.0, so this should be safe to
update.

Reply:
Good catch. I'll fix it 👍

Reply:
Looks like it's only `terser-webpack-plugin` that depends on `serialize-javascript` v1.x, so I'll just update the resolution from `**/serialize-javascript` to `**/terser-webpack-plugin/serialize-javascript`

Reply:
Fixed